### ISSUE TITLE
panic:FORK BOMB KILL SWITCH ACTIVATED

### ISSUE THREAD

**Issue by ekans** [teaxyz/cli#311]
Just running the command below in Fish:
```
echo '# tea *really is* a standalone binary' | tea --sync glow -
```

```
----------------------------------------------------->> attachment begin
Error: FORK BOMB KILL SWITCH ACTIVATED
    at default (file:///Users/builder/actions-runner/_work/cli/cli/src/app.exec.ts:10:28)
    at file:///Users/builder/actions-runner/_work/cli/cli/src/app.ts:54:27
<<----------------------------------------------------- attachment end
```

**Comment by tnxz** [comment]
refer to #308 and #306.

**Comment by mxcl** [comment]
wow, how could this fork bomb. Yeah #306 is the ticket.

**Comment by mxcl** [comment]
Can you do it again with:

```sh
echo '# tea *really is* a standalone binary' | tea -vvv --sync glow -
```

And also can I get a (santized) dump of `env`? Thank you.

**Comment by mxcl** [comment]
`which glow` too pls

**Comment by mxcl** [comment]
If you installed tea without the installer then the binaries attached to our releases were broken, fixed in v0.19.4

**Comment by mxcl** [comment]
Actually our cache invalidation for AWS cloud front was broken, but soon downloading the binaries without the installer will provide fixed binaries (v0.19.4)

**Comment by ekans** [comment]
I just tested and it works well!
Thanks.
